robot_controllers_interface/include/robot_controllers_interface/controller_manager.h

```diff
@@ -111,6 +111,10 @@ class ControllerManager
    */
   GyroHandlePtr getGyroHandle(const std::string& name);
 
+  /** @brief Load a controller from a class directly */
```

Comment:
suggest adding the params to the documentation as well - every other public function is fully described

Reply:
That was quick. Good point. I'll add a better explanation

Reply:
Added an explanation of the params. Do they make sense?